File: notifications/rest/services/update-service/update-service.6.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Notify.V1;

public class Example
{
    public static void Main(string[] args)
    {
        // Find your Account SID and Auth Token at twilio.com/console
        // To set up environmental variables, see http://twil.io/secure
        const string accountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
        const string authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");
        const string serviceSid = Environment.GetEnvironmentVariable("TWILIO_SERVICE_SID");

        TwilioClient.Init(accountSid, authToken);

        var service = ServiceResource.Update(
                          serviceSid,
                          friendlyName: "Another Awesome Service",
                          facebookMessengerPageId: "your_page_id",
                          messagingServiceSid: "your_twilio_messaging_service_sid"
                      );

        Console.WriteLine(service.FriendlyName);
    }
}

File: notifications/rest/services/update-service/update-service.6.x.rb
# Download the Ruby helper library from twilio.com/docs/libraries/ruby
require 'twilio-ruby'

# Get your Account Sid and Auth Token from https://www.twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

client = Twilio::REST::Client.new(account_sid, auth_token)

service = client.notify.v1.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')

service = service.update(
  friendly_name: 'Another Awesome Service',
  facebook_messenger_page_id: 'your_page_id',
  messaging_service_sid: 'your_twilio_messaging_service_sid'
)

puts service
